bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/BootstrapSliderCssResourceReference.java

```diff
@@ -0,0 +1,19 @@
+package de.agilecoders.wicket.extensions.slider.res;
+
+import org.apache.wicket.request.resource.PackageResourceReference;
+
+/**
+ * BootsTrapSliderResourceReference
+ */
+public class BootstrapSliderCssResourceReference extends PackageResourceReference {
+
+    private static final BootstrapSliderCssResourceReference instance = new BootstrapSliderCssResourceReference();
+
+    public BootstrapSliderCssResourceReference() {
+        super(BootstrapSliderCssResourceReference.class, "css/bootstrap-slider.min.css");
```

Comment:
Usually this should be the non-min CSS file. In production mode Wicket uses the min.css automatically.